apps/demo/src/styles.scss

```
@@ -401,6 +401,50 @@
+    --ngx-signal-form-fieldset-message-margin-bottom: 0.25rem;
+    --ngx-signal-form-fieldset-legend-padding: 0;
+    --ngx-signal-form-fieldset-legend-font-size: 0.875rem;
+    --ngx-signal-form-fieldset-legend-line-height: 1.25rem;
+    --ngx-signal-form-fieldset-legend-font-weight: 400;
+    --ngx-signal-form-fieldset-color: var(--color-text-muted);
+    --ngx-signal-form-fieldset-muted-color: var(--color-text-muted);
+  }
+
+  .choice-group-fieldset__legend {
+    @apply block text-sm font-normal tracking-wide;
+    padding-inline-start: 0.125rem;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)